scenario1/experiment_orchestrator.py

```diff
@@ -94,7 +94,7 @@ def start_vllm_server_client(benchmark_config, benchmark_name, k_client, mode, m
                     {
                         'name': 'vllm-server',
                         'image': "vllm/vllm-openai:v0.10.0",
-                        'command': ['vllm', 'serve'],
+                        'command': ['python3', '-m', 'vllm.entrypoints.openai.api_server'],
```

Comment:
This is already the default command for the image, so we don't need to include it. But I'm fine with it here for clarity.